Disable usage of pkg-config for OpenSSL to avoid linking two different versions.
See PRs: 258345, 271495

--- lib/tls/CMakeLists.txt.orig	2023-05-19 06:48:43 UTC
+++ lib/tls/CMakeLists.txt
@@ -264,10 +264,10 @@ if (LWS_WITH_SSL)
 		if (NOT OPENSSL_FOUND AND NOT LWS_WITH_BORINGSSL)
 			# TODO: Add support for STATIC also.
 			if (NOT LWS_PLAT_FREERTOS)
-				find_package(PkgConfig QUIET)
-				pkg_check_modules(PC_OPENSSL openssl QUIET)
+				#find_package(PkgConfig QUIET)
+				#pkg_check_modules(PC_OPENSSL openssl QUIET)
 				find_package(OpenSSL REQUIRED)
-				list(APPEND OPENSSL_LIBRARIES ${PC_OPENSSL_LIBRARIES})
+				#list(APPEND OPENSSL_LIBRARIES ${PC_OPENSSL_LIBRARIES})
 				set(OPENSSL_LIBRARIES ${OPENSSL_LIBRARIES} PARENT_SCOPE)
 			endif()
 			set(OPENSSL_INCLUDE_DIRS "${OPENSSL_INCLUDE_DIR}")
